CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file. This
 change log follows the conventions of
 [keepachangelog.com](http://keepachangelog.com/).
 
+## [3.6.1]
+- deprecate the matcher-combinators.midje ns and functions in it
```

Comment:
Fun to imagine this resulting in tens of thousands of new "deprecated" lint warnings in the test suites that Nubank still hasn't migrated away from Midje. Maybe this will help stoke the fire :P

Reply:
Nobody cares about deprecation warnings except @acamargo and me.